pages/osx/brightness.md

```diff
@@ -0,0 +1,15 @@
+# brightness
+
+> Set and obtain the brightness level of all internal and certain external displays.
+
+- Set 100% brightness:
```

Comment:
Perhaps this will read better?

```suggestion
- Set the brightness to 100%:
```

The same goes for the next example down.

Reply:
You're right. That sounds better.